tests/spark/test_text_operations.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import ast
+import json
+from textwrap import dedent
+
+import pytest
+# Import Operations to test
+from juicer.spark.text_operation import TokenizerOperation
+
+from juicer.spark.etl_operation import RandomSplit, Sort, Distinct, \
+    SampleOrPartition, AddRows, Intersection, Difference, Join, Drop, \
+    Transformation, Select, Aggregation, Filter, CleanMissing, AddColumns
+from tests import compare_ast, format_code_comparison
+
+
+def debug_ast(code, expected_code):
+    print
+    print code
+    print '*' * 20
+    print expected_code
+    print '*' * 20
+
+
+# Test TokenizerOperation
+
+def test_tokenizeroperation_type_simple():
+
+    # TYPE_PARAM = 'type'
+    # ATTRIBUTES_PARAM = 'attributes'
+    # ALIAS_PARAM = 'alias'
+    # EXPRESSION_PARAM = 'expression'
+    # MINIMUM_SIZE = 'min_token_length'
+    # TYPE_SIMPLE = 'simple'
+    # TYPE_REGEX = 'regex'
+
+    params = {
+        'type' : 'simple',
+        'attributes': 'col',
+        'alias': 'col_alias'
+    }
```

Comment:
Instead of using string constants, such as 'type', 'attributes', etc, import TokenizerOperation and use TokenizerOperation.TYPE_PARAM, TokenizerOperation.ATTRIBUTES_PARAM.